app/qml/form/MMPreviewDrawer.qml

```diff
@@ -249,6 +250,7 @@ Item {
                 width: parent.width
 
                 text: model.Value
+                textFormat: Text.PlainText
```

Comment:
As @VitorVieiraZ suggested, I would not change this here - I'd leave the option to pass in html, for the sake of links :)

Reply:
Other places look good to me!